Formula/e/enet.rb

```diff
@@ -26,14 +26,8 @@ class Enet < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "f0531f7c9cb6c994d6a38b07ac6e303f81a8a4194dd7daaa6d6aec1d25155afb"
   end
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
-    sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
-  end
-
   def install
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    system "./configure", *std_configure_args.reject { |s| s["--disable-debug"] }
```

Comment:
Would it be worth a comment for why we need to drop `--disable-debug` on this one?

Reply:
it is just a `configure` step warning, does not hurt to keep it either though. What do you prefer?

Reply:
Ah I think I understand, it's just you preserving the same `configure` args that were being given previous to the change? That seems fine too. I was thinking it was needed in order to support something in the build or functionality of the tool. But now I see there are maybe 10-or-so other formula which also have the same thing, maybe it's for the same reasons.